azerothcore/azerothcore-wotlk
# fix(DB/conditions): Fix Quest Kibler's Exotic Pets

You were able to complete the quest anywhere without the required creature being around you.





## CHANGES PROPOSED:
-  Add a condition to the spell

## ISSUES ADDRESSED:
- Closes #582



## TESTS PERFORMED:
- Executed SQL
-Tried to complete quest anywhere in the world
- Teleported near needed creature and watched it work



## HOW TO TEST THE CHANGES:

.quest add 4729
attempt to use added item "it should not work"

now do this:
.go c 40169
use item again

## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
